packages/playground/website/src/components/open-site-manager-button/index.tsx

```diff
@@ -0,0 +1,32 @@
+import css from './style.module.css';
+
+export function OpenSiteManagerButton() {
+	const onClick = () => {
+		// TODO use a shared function for switching views
+		const url = new URL(window.location.toString());
+		url.searchParams.set('view', 'manager');
+		window.location.assign(url.toString());
```

Comment:
Let's tie this URL param with the redux state, e.g. this could be `dispatch( setRoute( '/site-manager' ) );` and the reducer would do `pushState()` to eventually end up on `/?route=/site-manager`. On the initial load, the state would be populated from the search params, and everything in the app would use the redux store for state management – it's already easily observable and available across the board.

Reply:
I switched to the Navigator component, so now it matches what the site editor does. This also removed the reloads.